wiki/Help_Centre/en.md

```diff
@@ -660,14 +710,6 @@ You can always send a single reply to your currently open ticket asking about th
 
 In rare circumstances, support tickets from chronically offending users may be deliberately ignored, though the team will **always** tell you when they will no longer be responding to you first. If you have not received a notification like this, you do not need to worry about it!
 
-#### My user page contents are missing!
```

Comment:
what's the rationale behind deleting this?

Reply:
Sorry for any confusion, this one was moved to line ~773 and changed to: My Me! page contents are missing! Because thats the correct name for the section on an osu! profile.